package/gluon-respondd/luasrc/lib/gluon/upgrade/400-respondd-firewall

```diff
@@ -33,4 +33,24 @@ uci:section('firewall', 'rule',  'mesh_respondd_ll', {
 	target = 'ACCEPT',
 })
 
+uci:section('firewall', 'rule',  'mesh_respondd_siteprefix', {
+	name = 'mesh_respondd_siteprefix',
+	src = 'mesh',
+	src_ip = site.prefix6,
+	dest_port = '1001',
+	proto = 'udp',
+	target = 'ACCEPT',
+})
+
+if site.babel_mesh.prefix then
+	uci:section('firewall', 'rule',  'mesh_respondd_babelmeshprefix', {
+		name = 'mesh_respondd_babelmeshprefix',
+		src = 'mesh',
+		src_ip = site.babel_mesh.prefix,
+		dest_port = '1001',
+		proto = 'udp',
+		target = 'ACCEPT',
+	})
+end
```

Comment:
This should be moved to the Babel PR (`site.babel_mesh.prefix` doesn't even have a site check at the moment).

Reply:
done.